### PR TITLE
made rough homepage

### LINKED ISSUE #1: Create the homepage.
The homepage needs to be very minimalistic. 

- Title
- Significant input field for URL.
- Get report button.

### PR DESCRIPTION
**Pull request by sarthakkundra**
Created a landing page with :-  

- Dark / Light Mode
- Search Field
Screenshot 2020-12-27 at 6 26 16 PM

P.S. The text in the animation is just a placeholder. Wanted to make something like [this](https://vercel.com/) but CSS :)

Closes #1

### CODE REVIEW

File: components/Toggle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import clsx from 'clsx';
+
+import styles from './index.module.css';
+
+const index = () => {
+    const {  colorMode, toggleColorMode } = useColorMode();
+    return (
+        <div className={clsx(styles.right)}>
+             <Button onClick={toggleColorMode}>
+                {colorMode === "light" ? <img src={Dark} className={clsx(styles.icon)} /> : <img src={light} className={clsx(styles.icon)} />}
```
Comment (saphal1998) [inline components/Toggle/index.tsx:14]: @sarthakkundra 

Small nits: 
1) Can you make "light" and "dark" an enum? Don't want to compare strings here.

2) The casing of the svgs is a bit weird, it's `Dark` but it's also `light`. 

Like I said, small nits, feel free to address.

File: custom.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
+    const content: any;
+    export default content;
+  }
```
Comment (saphal1998) [inline custom.d.ts:4]:
```suggestion
declare module '*.svg' { 
    const content:  React.FunctionComponent<React.SVGAttributes<SVGElement>>; 
    export default content;
}
```

**Review by saphal1998 — APPROVED** [review]
Small nits but else LGTM🎉

**Comment by Saket-Upadhyay** [comment]
Had to merge to resolve the issue with API branch

**Pull request merged**